tests/src/python/test_provider_postgres.py

```diff
@@ -2535,7 +2535,7 @@ def testFeatureCountEstimatedOnTable(self):
         self.assertTrue(vl.isValid())
         self.assertTrue(vl.featureCount() > 0)
         vl.setSubsetString('"pk" = 3')
-        self.assertEqual(vl.featureCount(), 1)
+        self.assertTrue(vl.featureCount() > 0)
```

Comment:
"assertGreaterEqual" is better here -- it will give more useful output if the test fails